### DISCUSSION TITLE
Display license in installer with Agree / Quit buttons

### DISCUSSION THREAD

**Discussion by TRWoodland** [marcelotduarte/cx_Freeze#1871] [Ideas]
Would it be possible to display a license to the installer with Agree/Quit buttons?

I think it would need to be added to the "InstallUISequence" list in line 530 of command/_bdist_msi.py but I'm not sure.

Thanks

**Comment by marcelotduarte** [comment]
Sure, I accept a PR.

**Reply by ntindle** [reply]
Hey! Any pointers on how this could be done?

**Reply by marcelotduarte** [reply]
The end of InstallUISequence is [here](https://github.com/marcelotduarte/cx_Freeze/blob/main/cx_Freeze/command/bdist_msi.py#L204)
In the cx_Freeze docs has this link: https://docs.microsoft.com/en-us/windows/win32/msi/windows-installer-portal
Some other links:
https://learn.microsoft.com/en-us/windows/win32/msi/using-a-sequence-table
https://learn.microsoft.com/en-us/windows/win32/msi/dialog-boxes

**Reply by ntindle** [reply]
Thanks. Any other things you think need looked at while I’m in there? No promises but I can try if work agrees we need it

**Reply by ntindle** [reply]
#2472

**Comment by marcelotduarte** [comment]
[Release 7.2.0](https://github.com/marcelotduarte/cx_Freeze/releases/tag/7.2.0) is out!
[Documentation](https://cx-freeze.readthedocs.io)
